logging/log.go

```diff
@@ -400,6 +404,12 @@ func (l logger) GetTelemetryURI() string {
 	return telemetryConfig.URI
 }
 
+// GetTelemetryRemoteEnabled returns true if remote logging is enabled
+// This is decided by Enable parameter in logging.config
+func (l logger) GetTelemetryRemoteEnabled() bool {
```

Comment:
It's not about remote logging, it's purely about sending events ( and maybe a subset of the log messages ) to the telemetry server. A better name could be "GetTelemetryUploadingEnabled"